<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTransactionDialog</class>
 <widget class="QDialog" name="AddTransactionDialog">
  <layout class="QFormLayout">

   <item row="0" column="0">
    <widget class="QLabel"><property name="text"><string>–î–∞—Ç–∞</string></property></widget>
   </item>
   <item row="0" column="1"><widget class="QDateEdit" name="dateEdit"/></item>

   <item row="1" column="0">
    <widget class="QLabel"><property name="text"><string>–°—á—ë—Ç</string></property></widget>
   </item>
   <item row="1" column="1"><widget class="QComboBox" name="cmbAccount"/></item>

   <item row="2" column="0">
    <widget class="QLabel"><property name="text"><string>–ö–∞—Ç–µ–≥–æ—Ä–∏—è</string></property></widget>
   </item>
   <item row="2" column="1"><widget class="QLineEdit" name="edtCategory"/></item>

   <item row="3" column="0">
    <widget class="QLabel"><property name="text"><string>–¢–∏–ø –æ–ø–µ—Ä–∞—Ü–∏–∏</string></property></widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="operationType" native="true">
     <layout class="QHBoxLayout">
      <item><widget class="QRadioButton" name="rbtnExpense"><property name="text"><string>–†–∞—Å—Ö–æ–¥</string></property><property name="checked"><bool>true</bool></property></widget></item>
      <item><widget class="QRadioButton" name="rbtnIncome"><property name="text"><string>–î–æ—Ö–æ–¥</string></property></widget></item>
     </layout>
    </widget>
   </item>

   <item row="4" column="0">
    <widget class="QLabel"><property name="text"><string>–°—É–º–º–∞</string></property></widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="spinAmount">
     <property name="decimals"><number>2</number></property>
     <property name="maximum"><double>999999999.00</double></property>
     <property name="value"><double>0.00</double></property>
    </widget>
   </item>

   <item row="5" column="0">
    <widget class="QLabel"><property name="text"><string>–ö–æ–º–º–µ–Ω—Ç–∞—Ä–∏–π</string></property></widget>
   </item>
   <item row="5" column="1"><widget class="QLineEdit" name="edtComment"/></item>

   <!-- üîΩ –ù–æ–≤—ã–π –±–ª–æ–∫ –¥–ª—è —Ñ–æ—Ç–æ -->
   <item row="6" column="0">
    <widget class="QLabel"><property name="text"><string>–§–æ—Ç–æ</string></property></widget>
   </item>
   <item row="6" column="1">
    <widget class="QWidget" native="true">
     <layout class="QHBoxLayout">
      <item><widget class="QPushButton" name="btnAddPhoto"><property name="text"><string>–î–æ–±–∞–≤–∏—Ç—å —Ñ–æ—Ç–æ</string></property></widget></item>
      <item><widget class="QPushButton" name="btnOpenPhoto"><property name="text"><string>–û—Ç–∫—Ä—ã—Ç—å —Ñ–æ—Ç–æ</string></property></widget></item>
     </layout>
    </widget>
   </item>

   <item row="7" column="1">
    <widget class="QLabel" name="lblPhotoPath">
     <property name="text"><string>–§–æ—Ç–æ –Ω–µ –≤—ã–±—Ä–∞–Ω–æ</string></property>
    </widget>
   </item>

   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons"><set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set></property>
    </widget>
   </item>

  </layout>
 </widget>
</ui>
